web/cobrands/bristol/assets.js

```diff
@@ -24,7 +33,41 @@ var options = {
         asset_id: 'COD_ASSET_ID',
         usrn: 'COD_USRN'
     }
-};
+});
+
+var parkOptions = $.extend(true, {}, base_options, {
+    wfs_url: 'https://tilma.staging.mysociety.org/mapserver/bristol',
+    wfs_feature: "parks",
+    asset_type: 'area',
+    asset_id_field: 'SITE_CODE',
+    srsName: "EPSG:3857",
+    stylemap: park_style,
```

Comment:
this needs to be a stylemap, see other examples. It's breaking the tests, i'm afraid.